src/Ocelot/LoadBalancer/LoadBalancers/LoadBalancerHouse.cs

```diff
@@ -22,8 +22,6 @@ public Response<ILoadBalancer> Get(DownstreamRoute route, ServiceProviderConfigu
 
                 if (_loadBalancers.TryGetValue(route.LoadBalancerKey, out var loadBalancer))
                 {
-                    loadBalancer = _loadBalancers[route.LoadBalancerKey];

```

Comment:
@raman-m I removed this line, the `out` var already contained the correct reference.